ci/php.yml

```diff
@@ -0,0 +1,20 @@
+name: PHP Composer CI
```

Comment:
Let's remove the word `CI` based on the updated guidelines we are trying to merge in #156. It probably makes sense to keep using the word `Composer` to show that we are using composer.